package.json

```diff
@@ -228,6 +235,12 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "stripeInstallCLIView",
+        "contents": "Welcome! The Stripe VS Code extension requires the Stripe CLI.\n[Install Stripe CLI](https://stripe.com/docs/stripe-cli#install)\n[Set a custom path to Stripe CLI](command:workbench.action.openSettings?%5B%22stripe.cliInstallPath%22%5D)\nLearn more about [how to use the Stripe extension for Visual Studio Code](https://stripe.com/docs/stripe-vscode)."
```

Comment:
In case you missed it, Tomer had some comments in the vscode channel about making this message a bit softer with regards to it only being required for many features (as opposed to all).